tests/acceptance/infinity-test.js

```diff
@@ -20,7 +20,6 @@ module('Acceptance | infinity-scrollable', function(hooks) {
     await waitFor('.infinity-scrollable.inactive');
 
     assert.equal(findAll('.infinity-svg').length, 20);
-    assert.equal(findAll('.infinity-scrollable.inactive').length, 1, 'component is inactive after fetching more data');
```

Comment:
I *think* this is a race condition.  For one, this fails randomly on diff builds.  Second, it wouldn't proceed from the `await waitFor` unless it didn't find it as inactive.

Passes consistently locally; however, not up on CI.